server/router.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const methodOverride = require('method-override');
+const db = require('./db');
+
+const router = express.Router();
+
+let movieCount;
+
+router.use(bodyParser.urlencoded({ extended: true }));
+router.use(methodOverride('_method'));
```

Comment:
Nice solution here to `<form>` not allowing anything but `POST` and `GET`? Would be curious to know exactly what this is doing.